packages/visual-editing/package.json

```diff
@@ -164,29 +168,44 @@
     "@repo/package.config": "workspace:*",
     "@repo/visual-editing-helpers": "workspace:*",
     "@sanity/client": "6.21.2",
+    "@sanity/color": "^3.0.6",
+    "@sanity/demo": "^2.0.0",
     "@sanity/pkg-utils": "6.9.3",
     "@sanity/ui": "2.8.8",
+    "@storybook/addon-essentials": "^8.2.9",
+    "@storybook/addon-interactions": "^8.2.9",
+    "@storybook/addon-links": "^8.2.9",
+    "@storybook/addon-themes": "^8.2.9",
+    "@storybook/react": "^8.2.9",
+    "@storybook/react-vite": "^8.2.9",
+    "@storybook/test": "^8.2.9",
     "@sveltejs/adapter-auto": "^3.2.3",
     "@sveltejs/kit": "^2.5.21",
     "@sveltejs/package": "^2.3.3",
+    "@tailwindcss/typography": "^0.5.14",
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
     "@typescript-eslint/eslint-plugin": "^7.18.0",
     "@typescript-eslint/parser": "^7.18.0",
+    "add": "^2.0.6",
```

Comment:
Was this added intentionally?

Reply:
No! I think I ran something like `pnpm add npx add` by accident 🤦